cursor.go

```diff
@@ -39,6 +39,7 @@ type CursorConfig struct {
 	ColumnsToSelect []string
 	BuildSelect     func([]string, *TableSchema, uint64, uint64) (squirrel.SelectBuilder, error)
 	BatchSize       uint64
+	BatchSizeMap    map[string]map[string]uint64
```

Comment:
nit: `BatchSizeOverrides` maybe?

Reply:
...and keep it in sync with the config variable name (without the `DataIteration` prefix, like `BatchSize`)

Reply:
ok I agree, how about `BatchSizePerTableOverride`